package.nls.fr.json

```diff
@@ -0,0 +1,25 @@
+{
+  "command.regenerateIcons.title": "Appliquer Icônes Personnalisation",
```

Comment:
Sis also expressed a reservation on the translation but I insisted on keeping them as minimal as possible. She told that even so the meaning gets through.